### PR TITLE
[WIP] Cursor visibility

### PR DESCRIPTION
**Pull request by nifker**
Currently missing OSX implementation, would appreciate if someone could do it.

### CODE REVIEW

**Comment by emoon** [comment]
Found this. Should be easy to implement https://stackoverflow.com/questions/6350942/how-to-hide-the-mouse-cursor

**Comment by nifker** [comment]
@emoon Ive seen it recently too but I cant compile it on my machine(didnt got cross-compilation to work for now).

**Comment by emoon** [comment]
The CI will compile the code for you if I still want to implement it. It shouldn't be many lines of code to get it working.

**Comment by nifker** [comment]
@emoon on "[NSCursor hide];" - is NSCursor an object and hide the method?

**Comment by emoon** [comment]
yes, `NSCursor` here is a global `object` and hide/unhide are two methods

File: src/native/macosx/MacMiniFB.m [hunk 1]
```diff
@@ -423,6 +423,16 @@ void mfb_set_title(void* window, const char* title)
 	[win setTitle: ns_title];
 }
 
+void mfb_set_cursor_visibility(void *window, bool visibility)
+{
+	if (visibility){
+		[NSCursor show]
```
Comment (emoon) [inline src/native/macosx/MacMiniFB.m:429]: should be `NCursor unhide];`

File: src/native/macosx/MacMiniFB.m [hunk 2]
```diff
@@ -423,6 +423,16 @@ void mfb_set_title(void* window, const char* title)
...
+	}
+	else{
+		[NSCursor hide]
```
Comment (emoon) [inline src/native/macosx/MacMiniFB.m:432]: forgot a `;` after the method

**Comment by emoon** [comment]
Thanks! :)

**Pull request merged**